core/src/main/java/feign/Client.java

```diff
@@ -88,7 +88,7 @@ HttpURLConnection convertAndSend(Request request, Options options) throws IOExce
       connection.setConnectTimeout(options.connectTimeoutMillis());
       connection.setReadTimeout(options.readTimeoutMillis());
       connection.setAllowUserInteraction(false);
-      connection.setInstanceFollowRedirects(true);
+      connection.setInstanceFollowRedirects(options.isFollowRedirects());
```

Comment:
Won't this impact other clients too?

Reply:
The other clients would need to be updated to use the option.

Reply:
I made the option default to true, so the previous clients will keep the previous behavior.

You have to explicitely set the option to false to have the new behavior.

Reply:
This change need to be included to the okhttp and ribbon
https://github.com/OpenFeign/feign/blob/master/okhttp/src/main/java/feign/okhttp/OkHttpClient.java
https://github.com/OpenFeign/feign/blob/master/ribbon/src/main/java/feign/ribbon/RibbonClient.java

Reply:
@velo I have added implementations for Ribbon, OkHttp and LbClient as well (since unit tests of Ribbon use it too).